src/webhook/handlers/askingArticleSubmissionConsent.js

```diff
@@ -59,6 +62,9 @@ export default async function askingArticleSubmissionConsent(params) {
       CreateArticle.id
     );
 
+    // Create new session, make article submission button expire after submitting
+    data.sessionId = Date.now();
```

Comment:
This is brilliant 👍 

Cannot think of edge cases here. Let's test this in Wednesday.